.github/workflows/build.yml

```diff
@@ -0,0 +1,52 @@
+name: Build toolchain
+
+on:
+  push:
+    branches:
+      - tailscale
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+    - name: Set up Go
+      uses: actions/setup-go@v1
+      with:
+        go-version: 1
+      id: go
+
+    - name: Checkout
+      uses: actions/checkout@v2
+
+    - name: Build toolchain
+      run: cd src && ./make.bash
```

Comment:
probably worth running an `all.bash`

Reply:
Or maybe make.bash here is fine, but all.bash on PRs?  (which we could start using)

Reply:
Done. all.bash takes 12min on github, so I'd stripped it down to make.bash while iterating.